src/languages/es.ts

```diff
@@ -2051,7 +2051,7 @@ export default {
             exportInvoicesDescription: 'Las facturas se exportarán a esta cuenta en QuickBooks Online.',
             exportCompanyCardsDescription: 'Establece cómo se exportan las compras con tarjeta de empresa a QuickBooks Online.',
             account: 'Cuenta',
-            accountDescription: 'Esta es la cuenta elegida para contabilizar la compensación de la entrada de diario de cada informe.',
+            accountDescription: 'Esta es la cuenta elegida para contabilizar la compensación de asiento contable de cada informe.',
```

Comment:
```suggestion
            accountDescription: 'Esta es la cuenta elegida para contabilizar la compensación del asiento contable de cada informe.',
```

Reply:
aw man I changed this to `del` but then second guessed myself